product_offer/views/product_template_views.xml

```diff
@@ -485,7 +485,54 @@
         id="menu_product_template_zh_availability_action"
         parent="base.menu_product" sequence="10" />
 
+    <!-- Partner Stock Offer: derived from Product Offer (EN)  availability-->
+    <record id="product_template_en_partner_offer_kanban" model="ir.ui.view">
+        <field name="name">Product.template.product.partner.offer.kanban</field>
+        <field name="model">product.template</field>
+        <field name="arch" type="xml">
+            <kanban>
+                <field name="type"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div class="oe_kanban_vignette oe_semantic_html_override">
+                            <a type="open"><img t-att-src="kanban_image('product.template', 'image_small', record.id.value)" class="oe_kanban_image"/></a>
```

Comment:
Did the `change_image_resolution` include this view yet?

Reply:
True.